docs/modelserving/v1beta1/llm/huggingface/vllm_runtime.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  name: huggingface
+spec:
+  predictor:
+    containers:
+    - args:
+      - --model_name=llama2
+      - --model_id=meta-llama/Llama-2-7b-chat-hf
+      image: kserve/huggingfaceserver:latest
+      name: kserve-container
+      resources:
+        limits:
+          cpu: "6"
+          memory: 24Gi
+          nvidia.com/gpu: "1"
+        requests:
+          cpu: "6"
+          memory: 24Gi
+          nvidia.com/gpu: "1"
```

Comment:
Let's use the huggingface runtime with `modelFormat` field.

Reply:
Updated